src/datastore/postgres-store.ts

```diff
@@ -528,19 +528,25 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     return { found: true, result: tx };
   }
 
-  async getTxList(count = 50) {
+  async getTxList({ limit, offset }: { limit: number; offset: number }) {
+    const total = await this.pool.query<TxQueryResult>(`
+      SELECT *
```

Comment:
This query will run into scalability problems very quickly. Should be something like `SELECT count(*) ...`

In general `SELECT *` should be avoided

Reply:
Done,